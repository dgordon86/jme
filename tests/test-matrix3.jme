var c;

main()
{
 c = [1;2;3];
 a = [1,2,3;4,8,"hello"];
 print( a[0][0] );
 print( a[0][1] );
 
 b = a[1][1];
 print(b);
 
 print(a[1][2]);
 
 print(c[0][0]);
 print(c[1][0]);
 print(c[2][0]); 
}

