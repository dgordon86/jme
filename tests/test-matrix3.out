1
2
8
hello
1
2
3